launchable/grammer.lark

```diff
@@ -0,0 +1,14 @@
+?start: functions
+
+?functions: sort_by_most_likely_to_fail
+    | subset_by_duration
+
+sort_by_most_likely_to_fail: "sortByMostLikelyToFail" "(" (all_tests|functions) ")"
+subset_by_duration: "subsetByDuration" "(" percentage "," (all_tests|functions) ")"
```

Comment:
The idiom of BNF is to create a rule around types, thus "list" is a better rule name than "functions":
```
start : list
list : "allTests"
     | "sortByMostLikelyToFail" "(" list ")"
     | "subsetByDuration" "(" percentage ',' list ")"
...
```